src/components/Service.jsx

```diff
@@ -30,11 +37,27 @@ const ScreenSplit = ({ data }) => {
 
   const mapDisplay = () => {
     if (collapse) {
-      return <Button data-cy="toggle-map-visibility-button" size="small" color="secondary" onClick={()=>setCollapse(false)}>Show on map</Button>;
+      return (
+        <Button
+          data-cy="toggle-map-visibility-button"
+          size="small"
+          color="secondary"
+          onClick={() => setCollapse(false)}
+        >
+          Show on map
+        </Button>
+      );
     } else {
       return (
         <Box component="div" style={styles.map}>
-          <Button data-cy="toggle-map-visibility-button" size="small" color="secondary" onClick={()=>setCollapse(true)}>Collapse map</Button>
+          <Button
+            data-cy="toggle-map-visibility-button"
+            size="small"
+            color="secondary"
+            onClick={() => setCollapse(true)}
+          >
+            Collapse map
+          </Button>
```

Comment:
It was there before but we can refactor it. replace to `setCollapse(!collapse)`, and button content to ternary `{collapse ? 'Show on map' : 'Collapse map'}`

then we dont need if statement at all

Reply:
fixed